.travis.yml

```diff
@@ -348,7 +348,7 @@ jobs:
         - sudo bash .setup/travis/setup.sh
         - sudo bash .setup/travis/setup_test_suite.sh
       script:
-        - sudo python3 /usr/local/submitty/test_suite/integrationTests/run.py
+        - travis_wait 45 sudo python3 /usr/local/submitty/test_suite/integrationTests/run.py
```

Comment:
Did you mean to add this? It seems like this should not be in this pr

Reply:
Trying a thing to see if it fixes the 10m0s timeout bug
(Derek discovered this magic Travis hack)

Reply:
Ok, I just wanted to make sure this code was not added by mistake.